cli/CHANGELOG.md

```diff
@@ -1,16 +1,16 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
 ## 13.6.4
 
-_Released 1/30/2024 (PENDING)_
-
-**Bugfixes:**
-
-- Fixed an issue with capturing assets for Test Replay when service workers are registered in Cypress support files. This issue would cause styles to not render properly in Test Replay. Fixes [#28747](https://github.com/cypress-io/cypress/issues/28747).
+_Released 1/30/2024_
 
 **Performance:**
 
 - Fixed a performance regression from [`13.3.2`](https://docs.cypress.io/guides/references/changelog#13.3.2) where aborted requests may not correlate correctly. Fixes [#28734](https://github.com/cypress-io/cypress/issues/28734).
 
+**Bugfixes:**
+
+- Fixed an issue with capturing assets for Test Replay when service workers are registered in Cypress support files. This issue would cause styles to not render properly in Test Replay. Fixes [#28747](https://github.com/cypress-io/cypress/issues/28747).
+
 **Misc:**
```

Comment:
We typically miss these but looks like @cypress/webpack-dev-server had a release which means we will have a dep bump in the binary as well. see https://github.com/cypress-io/cypress/commit/f21b4a12a1ba1c1a7c7bc30b9a6da1142c78529a

Reply:
should we log that dep bump in the changelog here?